src/router/index.ts

```diff
@@ -85,4 +85,14 @@ router.beforeEach((to: RouteLocationNormalized,
 });
 
 
+/** Call `getRedirectResult` when app starts
```

Comment:
You might also call `getRedirectResult` here. `import router from "@/router";` is called in `src/main.ts`, in `src/views/auth/Login.vue` and wherever you want to redirect stuff. I.e. `getRedirectResult` would be called several times.